.github/workflows/check.yml

```diff
@@ -2,8 +2,11 @@ name: check
 
 on:
   pull_request:
+    paths:
+    - '.github/workflows/check.yml'
     paths-ignore:
     - 'Firestore/**'
```

Comment:
Oh, so it looks like `check` was already set up to run `check` on `check.yml` changes.

GHA failed with: `you may only define one of `paths` and `paths-ignore` for a single event`